src/Service/Gateway.php

```diff
@@ -98,8 +109,8 @@ public function finalize(
                $message ='Error during transaction';
                $message .= isset(current($order['transactions'])['reason']) ? ':'.current($order['transactions'])['reason'].'.' : '.';
                $message .= '<br> Please contact support.';
-            $this->helper->saveEMSLog($message, ['FILE' => __FILE__, 'FUNCTION' => __FUNCTION__, 'LINE' => __LINE__]);
-            print_r($message);exit;
+
+            throw new EmsPluginException($message);
             throw new CustomerCanceledAsyncPaymentException(
                 $transactionId,
                 (current($order['transactions'])['reason'])
```

Comment:
Remove this Exception we don't throwing it

Reply:
Done